client/src/components/Sidebar/Sidebar.tsx

```diff
@@ -233,6 +236,9 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = ({
                 hover:opacity-100
                 more-contrast:w-border-border-interactive-more-contrast-dark-bg
                 hover:more-contrast:w-border-border-interactive-more-contrast-dark-bg-hover`}
+              data-controller="w-kbd"
+              data-w-kbd-key-value="mod+["
+              data-w-kbd-scope-value="global"
```

Comment:
Same here, let's use the target approach, making it NOT the target when we are in mobile view.

```suggestion
              data-w-kbd-target={!isMobile ? 'element' : undefined}
```

And, let's set `aria-keyshortcuts` further up.

```ts
// ...
            <button
              onClick={onClickCollapseToggle}
              aria-label={gettext('Toggle sidebar')}
              aria-expanded={slim ? 'false' : 'true'}
              aria-keyshortcuts="["
              type="button"
// ...
```